From 5752aa3fcbf38df8b2afa728459642723083d214 Mon Sep 17 00:00:00 2001
From: Kai Liu <kai.liu@suse.com>
Date: Wed, 29 Jun 2022 20:21:01 +0800
Subject: [PATCH] Revert "Linux 5.10.127"
Patch-mainline: Never, revert stable version change
References: bsn#83
Modified-by-SEL: No

This reverts commit deb587b1a48d3ac4a29378fc238a4677944dff83.
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index e3eb9ba19f86..97c37c3ab0da 100644
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: GPL-2.0
 VERSION = 5
 PATCHLEVEL = 10
-SUBLEVEL = 127
+SUBLEVEL = 126
 EXTRAVERSION =
 NAME = Dare mighty things
 
-- 
2.35.1

